ledger/ledger-api-integration-tests/src/test/lib/scala/com/digitalasset/platform/PlatformApplications.scala

```diff
@@ -46,17 +46,30 @@ object PlatformApplications {
       "Max TTL's granularity is subsecond. Ledger Server does not support subsecond granularity for this configuration - please use whole seconds."
     )
 
-    def getLedgerId: String =
-      ledgerId.getOrElse(
-        throw new IllegalStateException("Attempted to access ledger ID, but none is configured."))
+    @SuppressWarnings(Array("org.wartremover.warts.StringPlusAny"))
+    def assertStaticLedgerId: String =
```

Comment:
Assertions can be implicit, I don't think we need to express them in the method name. How about using `staticLedgerId` instead?

Reply:
IMO it is better to have it explicit, since if someone calls into this function when `ledgerId` is not static, it is a bug in the code. It is the same style as prefixing partial or dangerous functions with `unsafe`.

Reply:
Ok, let’s leave it. My take is that as long as we have good tests and documented behaviour this should not matter and it just makes the code less fluent/readable.